cmd/antrea-controller/controller.go

```diff
@@ -307,44 +327,44 @@ func run(o *Options) error {
 	informerFactory.Start(stopCh)
 	crdInformerFactory.Start(stopCh)
 
-	go clusterIdentityAllocator.Run(stopCh)
+	goExec(clusterIdentityAllocator.Run, stopCh)
```

Comment:
my opinion here is that introducing `goExec` makes the code less readable and idiomatic, and that doing this for the sake of adding a unit test is not the right option.
cc @tnqn

Reply:
Good consideration and I haven’t got a better solution yet. @tnqn @luolanzone Do you have any idea?

Reply:
done